pkg/kudoctl/cmd/install/install.go

```diff
@@ -113,8 +111,20 @@ func installOperator(operatorArgument string, options *Options) error {
 		return errors.Wrapf(err, "failed to resolve package CRDs for operator: %s", operatorArgument)
 	}
 
+	return installCrds(crds, kc, options)
+}
+
+func installCrds(crds *repo.PackageCRDs, kc *kudo.Client, options *Options) error {
```

Comment:
this function conflates things IMO (at least using this function name.   It does the following:
1. installs CRDs
2. installs an instance
3. validates Crds

at this point it looks like `InstallPkgsAndValidateCrds`

Reply:
Instance is CRD, so point 1 and 2 are the same thing. I can rename it to `installAndValidateCrds` but I felt like the validation can be an implementation detail from caller perspective - I felt like the name does not have to reflect step by step what is the function doing.

If you feel strongly about this I can rename, my preference would be to keep the name simple. obviously the method has to return errors when it cannot install (there was some error, which is what the validation is doing)

Reply:
I think this is ok with how the code is structured right now. We could create a better interface for installation if we really wanted to, so each CRD knows how to install and validate itself and push that code out of here.